tools/check_format.py

```diff
@@ -85,16 +86,46 @@ def isBuildFile(file_path):
   return False
 
 
-def checkFileContents(file_path):
-  message = "%s has over-enthusiastic spaces:" % file_path
-  findSubstringAndPrintError('.  ', file_path, message)
+def headerSubdirDict():
+  subdir_dict = {}
+  for subdir in common.includeDirOrder():
+    subdir_dict[subdir] = True
+  return subdir_dict
```

Comment:
I would just make this `headerSubdirSet` and then this entire function becomes `return set(common.includeDirOrder())`.

Reply:
done -- actually that became simple enough I just did that in the const initializer.